src/lib/utils/accounts.ts

```diff
@@ -29,3 +33,47 @@ export const isValidAccountNumber = (
   if (accountNumber === null) return false
   return Number.isInteger(accountNumber) && accountNumber >= 0
 }
+
+export const isUtxoAccountWithAddresses = (
```

Comment:
Not sure what that naming refers to here - also wouldn't `addresses` always be populated?
Despite types saying it's optional - if you hit `/api/v1/account/<pubkey` (where pubkey may be a xpub or an `address_index` addy) you'll notice that `addresses` is always populated.

Reply:
Removed the optional, and renamed to `isUtxoAccount` in [dae8b45](https://github.com/shapeshift/web/pull/10928/commits/dae8b45dc9e90ec32cc489375fee088726e2a112)